website/sidebars.js

```diff
@@ -8,6 +8,14 @@
  */
 
 const {fbInternalOnly} = require('internaldocs-fb-helpers');
```

Comment:
Our internal website previewer is having trouble finding the tutorials because it doesn't know about the custom scripts :). I can look into whether it's possible to add custom commands to the builder, but in the mean time, could you try making the page external only?

To do so, you can import `fbContent` from `'internaldocs-fb-helpers'`, i.e. replace this line with:
```jsx
const {fbInternalOnly, fbContent} = require('internaldocs-fb-helpers');
```